common/config/rush/.npmrc

```diff
@@ -21,8 +21,5 @@
 registry=https://registry.npmjs.org/
 always-auth=false
 # No phantom dependencies allowed in this repository
-# Don't hoist in common/temp/node_modules
-public-hoist-pattern=

```

Comment:
Can we leave these on and resolve whatever issue is showing up instead of turning them off?